Assets/MixedRealityToolkit.Extensions/Sharing/SpatialAlignment/Common/SpatialCoordinateServiceBase.cs

```diff
@@ -84,7 +84,8 @@ protected void OnNewCoordinate(TKey id, ISpatialCoordinate spatialCoordinate)
             {
                 if (!isTracking)
                 {
-                    throw new InvalidOperationException("We aren't tracking, and shouldn't expect additional coordinates to be discovered.");
+                    UnityEngine.Debug.LogWarning("New coordinate discovered, but spatial coordinate service was not tracking");
```

Comment:
Yeah, this is interesting. Maybe we can do a silent return, or should we actually process since these were as part of previous detection.

I was to strict missing a clear race condition here, strangely I didn't hit it when testing.

Reply:
I think for now ill log this warning and return. It may be that we need to process this in the future, but right now it seems like processing old logic isn't necessary.